hbp_knowledge/chung2018.bel

```diff
@@ -0,0 +1,739 @@
+#######################
+# Document Properties #
+#######################
+SET DOCUMENT Copyright   = "Copyright © 2018 Fraunhofer Institute SCAI, All rights reserved."
+SET DOCUMENT Licenses    = "Private Ownership to Fraunhofer Institute SCAI"
```

Comment:
We're using `SET DOCUMENT Licenses    = "CC BY 4.0"` for everything in this repository